py-scripts/sandbox/cc_module_opensync.py

```diff
@@ -1,3 +1,4 @@
+# flake8: noqa
```

Comment:
missing shebang

Reply:
This script is in sandbox, but I can add a shebang if desired. This seems outside the scope of this PR, though

Reply:
sandbox, yeah don't care